### ISSUE TITLE
Ranger icons not showing properly

### ISSUE THREAD

**Issue by solomonxie** [francoiscabrol/ranger.vim#74]
I'm using the plugin [`alexanderjeurissen/ranger_devicons`](https://github.com/alexanderjeurissen/ranger_devicons) for Ranger.

![image](https://user-images.githubusercontent.com/14041622/52106554-35888000-262e-11e9-951d-62ec511180f1.png)

But when it comes to the `Ranger.vim`, icons are not showing properly. 

![image](https://user-images.githubusercontent.com/14041622/52106600-623c9780-262e-11e9-9272-a8fa334d4549.png)

Probably it's the `NerdFont` issue but I have no idea where to work on.

**Comment by neumachen** [comment]
Are you using OSX?

**Comment by solomonxie** [comment]
> Are you using OSX?

Yes i'm using MacOS 10.12 Sierra. My terminal is iTerm2 with Nerd font.

**Comment by neumachen** [comment]
@solomonxie it's most likely because your iTerm font is not using Nerd Font or your Nerd Font is not patched.

**Comment by solomonxie** [comment]
> @solomonxie it's most likely because your iTerm font is not using Nerd Font or your Nerd Font is not patched.

Thanks for reply, yes I guess it might be the answer. I'm using Nerd font on iTerm2, but except `Droid Sans Mono Nerd Font` I can't use any others for proper display.

![image](https://user-images.githubusercontent.com/14041622/52907355-34488b80-329b-11e9-92e9-4fd0aa58e960.png)

**Comment by windvalley** [comment]
I ran to the same problem, just to add `export LC_ALL="en_US.UTF-8"` to `~/.zshrc` to solve the problem.
